### ISSUE TITLE
Screen & update family-facing website pages for Lookit --> Children Helping Science transition

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1061]
# TL;DR

Ctrl+F Lookit -> CHS, but unfortunately probably not that simple!

# Narrative

As a new family, I should basically think of myself as a "CHS" family, and not see the "Lookit" name on web copy or emails.  

# Acceptance Criteria

- [ ] List of all affected URLs
- [ ] Melissa provides copy changes for all pages!
- [ ] Pages all updated/implemented

# Implementation Notes

Another CHS-transition issue that should see a coordinated release to production

**Comment by mekline** [comment]
Returning to this issue following some new decisions on naming - we'll continue with the "Children Helping Science, powered by Lookit" framing, where CHS refers to:

- The team running the website
- The family-facing website
- The umbrella term for any study posted to the search page

And Lookit refers to:

- The experiment runner engine
- The back-end, researcher-facing website & community infrastructure
- The "system" that powers CHS, from the family's perspective

This distinction (according to our lawyer) will mean that we *don't* need to rewrite consent forms or have labs sign new agreements - it's a "marketing" change on our side, and labs can continue to use the lookit.mit.edu links if their IRBs are more conservative.

**Comment by mekline** [comment]
In light of this, here are some proposals for the walkthrough you sent! I will probably need to run these by Laura, but to get them down as we're going: 

2. Sign up page (https://lookit.mit.edu/signup)
   `participant-signup.html`
   _"Ready to participate in a **Lookit** study?"_

   --> "Ready to participate in a study with Children Helping Science?"


4. (Will handle emails in the other issue)

5. Log in page (https://lookit.mit.edu/login)   `login.html`
   _"New to **Lookit**?"_

   --> "New to Children Helping Science?"

6. Study list page (https://lookit.mit.edu/studies)
   `accounts/forms.py`, `web/templatetags/web_extras.py`
   _"Show studies that are ... happening on **Lookit**"_ (drop-down option)

   ---> "Show studies happening..."here on the Lookit platform"/"on other websites"

   _"**Lookit** is growing! We are now showing links to outside studies along with those happening here on **Lookit**. "_

   --> Drop this message since external studies are no longer new, so the whole of that paragraph reads: "Use the tabs above to see activities you can do right now, or scheduled activities you can sign up for."

7. Study detail page (https://lookit.mit.edu/studies/<uuid>)
   `web/templates/web/study-detail.html`
   _"This study is being run outside of **Lookit**"_ (external studies only)

    --> Update the messages for both internal and external studies: 

   Internal:  This study runs here on the Lookit platform. Clicking to participate will make it possible for the researchers of this study to contact you, and will share your child's profile information with the researchers.

   External:  This study runs on another website, and clicking to participate will take you to that link. It will also make it possible for the researchers of this study to contact you, and will share your child's profile information with the researchers.


9. Studies history (https://lookit.mit.edu/studies/history)
   `accounts/forms.py`, `web/templates/web/studies-history.html`
   _"**Lookit** studies"_ (navigation tab label)
   _"You have not participated in any **Lookit** studies."_

   --> Leave the tab labels alone, and update the explainer text that appears below: 

   "Lookit studies run here on our platform, and usually include a video of your session! You can find information about your previous study sessions, watch these videos, and see comments left by researchers below."

   "Lookit studies run here on our platform, and usually include a video of your session!  You have not participated in any Lookit studies." 

   "External studies happen at other websites, and are listed below after you click the "Participate Now" button to follow a link. You can find information on the studies you have accessed and contact information for the researchers on a specific study below."

   "External studies happen at other websites, and are listed below after you click the "Participate Now" button to follow a link. You have not participated in any external studies."

10. See stub issue for FAQ at #1179 

11. See stub issue for Scientists page at #1180

13. See stub issue for Privacy at #1181


17. All page titles (shown in browser tab) 
   `web/templates/web/base.html`
    All titles end with " - **Lookit**", e.g. the studies page is called "Studies - Lookit"

   --> "PageTitle - Children Helping Science"

1. Resources (https://lookit.mit.edu/resources/)
   `web/templates/web/resources.html`, `web/static/js/resources.js`
   _"Visit **Children Helping Science** for an extensive list of other opportunities to participate in research about child development!"_

--> Delete this sentence (so SciStarter will be the only other link for now)

And I found one link to a specific CHS page. This link will break after the domain transfer, so we either need to add a publications page to our site or change/remove the link.
1. FAQ (https://lookit.mit.edu/faq/)
    `web/templates/web/faq.html`
    _"You can click &lt;a href = "https://childrenhelpingscience.com/publications" &gt;here&lt;/a&gt; to see some examples of scientific research published with data collected online with children."_

--> Great catch!! New Issue: #1182

**Comment by mekline** [comment]
@becky-gilbert please don't move forward on this until I have final signoff from Laura, but then we should close this issue & create new ones in the organization/chunks that make sense for development work.  Thank you again for mapping this out!!

**Comment by becky-gilbert** [comment]
@mekline I believe I've now made all of the changes you've requested in your [comment above](https://github.com/lookit/lookit-api/issues/1061#issuecomment-1518172665). I edited your comment to add the associated pull request at the bottom of each item, and those PRs are ready for your review 🙂 

The only item that I haven't addressed is the Privacy page:

> 10. See stub issue for Privacy at https://github.com/lookit/lookit-api/issues/1181

Because that issue (#1181) doesn't describe any specific changes. Did you want me to wait for you to add more info to that issue? Or should I go through the page myself to make sure that the "Lookit"/"CHS" wording is being used appropriately?
